get_experiment_data.R

```diff
@@ -97,26 +93,35 @@ option_list = list(
     make_option(
         c("-g", "--number-of-clusters"),
         action = "store",
-        default = NA,
-        type = 'integer',
-        help = "Number of clusters for marker gene file"
+        default = "inferred_cell_type_-_ontology_labels",
```

Comment:
I think we could clean this up a bit to make it more understandable. 

Rather than 'number of clusters', this could be a more generic 'cell grouping' or something. Then tell the user that integer values will be interpreted as referring to the automated clusterings.